test/test-examples-server-https-fail-checks.js

```diff
@@ -75,7 +75,8 @@ describe("Alexa App Server with Examples & HTTPS fail checking", function() {
     expect(testServer.httpsInstance).to.not.exist;
   });
 
-  it("fails to mount due to invalid port", function() {
+  // binding to port -1 seems to behave differently on versions of node < 6
+  xit("fails to mount due to invalid port", function() {
```

Comment:
Yep, it marks test as pending. See http://stackoverflow.com/questions/11518609/declare-pending-specs-tests-in-jasmine-or-mocha for example. It's a common BDD convention.